app/uk/gov/hmrc/securemessage/controllers/SecureMessageController.scala

```diff
@@ -40,19 +45,19 @@ class SecureMessageController @Inject()(
 
   def createConversation(client: String, conversationId: String): Action[JsValue] = Action.async(parse.json) {
     implicit request =>
-      withJsonBody[ConversationRequest] { conversationRequest =>
-        secureMessageService.createConversation(conversationRequest.asConversation(client, conversationId)).map {
-          case Right(_)                        => Created
-          case Left(error: SecureMessageError) => handleErrors(client, conversationId, error)
-        }
-      }.recover {
-        case error: Exception => handleErrors(client, conversationId, error)
+      client match {
+        case "cdcm" =>
```

Comment:
what happens if I pass "CDCM" instead of "cdcm"?

Reply:
thanks you, it will fail, Oleg's idea to make it more strict